README.md

```diff
@@ -16,7 +16,7 @@ Badness  Function
 Make sure that you have the following requirements:
 
   * `rustup` and `cargo` must be installed and in your `PATH`
-  * LLVM 8, 9, or 10 must be available (see https://crates.io/crates/llvm-ir, if you're using a version besides 9, you'll need to change the feature in Siderophile's Cargo.toml before you compile)
+  * LLVM 8, 9, or 10 must be available (see https://crates.io/crates/llvm-ir, if you're using a version besides 11, you'll need to change the feature in Siderophile's Cargo.toml before you compile)
```

Comment:
We don't or won't really test with LLVM 8/9/10, do we?

Maybe we should change this to:

> LLVM 11 is required. Older versions such as LLVM 8, 9 or 10 may work (see https://crates.io/crates/llvm-ir) but requires the `llvm-ir`'s package feature change in `Cargo.toml` before compiling Siderophile.

Reply:
Thanks, @disconnect3d. I made the change.